core/src/main/java/hudson/model/AbstractBuild.java

```diff
@@ -880,15 +880,12 @@ public EnvVars getEnvironment(TaskListener log) throws IOException, InterruptedE
         if (ws!=null)   // if this is done very early on in the build, workspace may not be decided yet. see HUDSON-3997
             env.put("WORKSPACE", ws.getRemote());
 
-        project.getScm().buildEnvVars(this,env);
+        project.getScm().buildEnvironment(this, env);
```

Comment:
Just as a reminder that we are now following the conventional rename for this sort of API change.